configs/NEUTRONRCF435/config.h

```diff
@@ -86,6 +86,8 @@
 
 #define CAMERA_CONTROL_PIN              PA0
 
+#define USE_ADC
+#define ADC_INSTANCE                    ADC1
```

Comment:
ADC1 is default when not defined.

https://github.com/betaflight/betaflight/blob/69999281c460c82a294de4465ca7b92a320663f5/src/main/drivers/adc.h#L28-L30